packages/terra-clinical-header/src/Header.jsx

```diff
@@ -35,34 +53,53 @@ const propTypes = {
    * A Boolean indicating if element is a subheader.
    */
   isSubheader: PropTypes.bool,
+
+  /**
+   * IDs should be used with 'aria-labelledby' to associate headings with the corresponding page area when needing to label regions.
+   *
+   * For an example, read [Accessibility Guide: Labeling Section Content](/components/terra-clinical-header/clinical-header/accessibility-guide#labeling-section-content) for additional information.
+   */
+  id: PropTypes.string,
 };
 
 const defaultProps = {
   title: '',
   startContent: null,
   endContent: null,
   isSubheader: false,
+  text: '',
 };
 
 const Header = ({
-  children, title, startContent, endContent, isSubheader, ...customProps
+  children, title, startContent, endContent, text, level, id, isSubheader, ...customProps
 }) => {
   const theme = useContext(ThemeContext);
+  if (title) {
+    // TODO: remove on the next major version bump
+    // eslint-disable-next-line no-console
+    console.warn('The `title` prop has been renamed to `text`. Please update all references of `title` prop to `text`.');
+  }
+  if (!level) {
+    // TODO: remove on the next major version bump
+    // eslint-disable-next-line no-console
+    console.warn('Default heading level may not appropriate has it would fail to convey context of heading in a site / application where it is used. Heading level should be set explicitly depending on the position of header in site / application to allow screen readers to identify headers consistently.');
+  }
```

Comment:
We should add unit tests for these to test if console.warn has the appropriate output.

Reply:
added jest tests : https://github.com/cerner/terra-clinical/pull/823/commits/2a0747c1873dccb3053cc005b59cdea87205589c#diff-17930a9789595a8fe280df0bac7f10c1ab42f97d088b819afada3036b66b3612